assets/sass/components/reader-revenue-manager/_googlesitekit-rrm-overlay-notification.scss

```
@@ -0,0 +1,29 @@
+/**
+ * Reader Revenue Manager overlay notification styles.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-overlay-notification {
+	&.googlesitekit-reader-revenue-manager-publication-approved-notification {
+		.mdc-button--raised {
+			font-weight: 500;
+		}
```

The button text and icon should have a gap of 6px, and we should use appropriate variable for the font weight:

```suggestion
		.mdc-button--raised {
			gap: 0 6px;
			font-weight: 500;
		}
```